Failed to install dependencies on ARM

Trying to ``./deps.sh`` on ARM:

```
Installing http://rocks.tarantool.org/luatest-0.5.4-1.all.rock

luatest 0.5.4-1 is now installed in /mnt/mmc-SD64G_0x016e52ea-part1/crud/.rocks (license: MIT)

Installing http://rocks.tarantool.org/luacheck-0.25.0-1.rockspec

Cloning into 'luacheck'...
remote: Enumerating objects: 6143, done.
remote: Total 6143 (delta 0), reused 0 (delta 0), pack-reused 6143
Receiving objects: 100% (6143/6143), 2.18 MiB | 4.41 MiB/s, done.
Resolving deltas: 100% (4029/4029), done.
Note: switching to 'ec42b3d5a44ec3b9419bab0c6955d64d3feb0f02'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

Failed to set locale to C.UTF-8
luacheck 0.25.0-1 is now installed in /mnt/mmc-SD64G_0x016e52ea-part1/crud/.rocks (license: MIT)

Failed to set locale to C.UTF-8
Failed to set locale to C.UTF-8
Failed to set locale to C.UTF-8
Installing http://rocks.tarantool.org/luacov-0.13.0-1.rockspec

Cloning into 'luacov'...
remote: Enumerating objects: 78, done.
remote: Counting objects: 100% (78/78), done.
remote: Compressing objects: 100% (62/62), done.
remote: Total 78 (delta 13), reused 49 (delta 9), pack-reused 0
Receiving objects: 100% (78/78), 56.82 KiB | 239.00 KiB/s, done.
Resolving deltas: 100% (13/13), done.
Note: switching to '637c48d59f722050d718d0c398f655bc7fe1707a'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

Failed to set locale to C.UTF-8
luacov 0.13.0-1 is now installed in /mnt/mmc-SD64G_0x016e52ea-part1/crud/.rocks (license: MIT)

Failed to set locale to C.UTF-8
Failed to set locale to C.UTF-8
Failed to set locale to C.UTF-8

Cloning into 'cluacov'...
remote: Enumerating objects: 65, done.
remote: Total 65 (delta 0), reused 0 (delta 0), pack-reused 65
Receiving objects: 100% (65/65), 39.87 KiB | 474.00 KiB/s, done.
Resolving deltas: 100% (26/26), done.
gcc -O2 -fPIC -I/usr/local/include/tarantool -c src/cluacov/deepactivelines.c -o src/cluacov/deepactivelines.o
In file included from src/cluacov/lj2/lj_obj.h:14,
                 from src/cluacov/deepactivelines.c:19:
src/cluacov/lj2/lj_arch.h:55:2: error: #error "No support for this architecture (yet)"
   55 | #error "No support for this architecture (yet)"
      |  ^~~~~
src/cluacov/lj2/lj_arch.h:294:2: error: #error "No target architecture defined"
  294 | #error "No target architecture defined"
      |  ^~~~~

Error: Build error: Failed compiling object src/cluacov/deepactivelines.o
```
By deleting this line, it's okay:
https://github.com/tarantool/crud/blob/f0445eaf53fa1ac279673f4c4b0ac95c007241f3/deps.sh#L9

```bash
$ uname -a
Linux ARM-FLEX 5.13.12-1-MANJARO-ARM #1 SMP Wed Aug 18 07:36:58 UTC 2021 aarch64 GNU/Linux
```

--- a/deps.sh
+++ b/deps.sh
@@ -16,7 +16,7 @@ $TTCTL rocks install luacheck 0.25.0
 $TTCTL rocks install luacov 0.13.0
 
 # cluacov, luacov-coveralls and dependencies
-$TTCTL rocks install https://raw.githubusercontent.com/mpeterv/cluacov/master/cluacov-scm-1.rockspec
+$TTCTL rocks install https://raw.githubusercontent.com/luarocks/cluacov/master/cluacov-scm-1.rockspec
 $TTCTL rocks install https://raw.githubusercontent.com/LuaDist/dkjson/master/dkjson-2.5-2.rockspec
 $TTCTL rocks install https://raw.githubusercontent.com/keplerproject/luafilesystem/master/luafilesystem-scm-1.rockspec
 $TTCTL rocks install https://raw.githubusercontent.com/moteus/lua-path/master/rockspecs/lua-path-scm-0.rockspec

--- a/test/helper.lua
+++ b/test/helper.lua
@@ -818,6 +818,12 @@ function helpers.skip_old_tarantool_cartridge_hotreload()
     t.skip_if(luatest_utils.version_ge(tarantool_version, luatest_utils.version(2, 0, 0))
           and luatest_utils.version_ge(luatest_utils.version(2, 5, 1), tarantool_version),
         "Cartridge hotreload tests stuck for vshard 0.1.22+ on Tarantool 2.2, 2.3 and 2.4")
+
+    -- Flaky tests for Tarantool 1.10.15.
+    -- s2-replica | E> ER_READONLY: Can't modify data because this instance is in read-only mode.
+    -- s2-replica | E> ApplyConfigError: Can't modify data because this instance is in read-only mode.
+    t.skip_if(luatest_utils.version_ge(luatest_utils.version(2, 0, 0), tarantool_version),
+        "Flaky tests for Tarantool 1.10.15")
 end
 
 function helpers.build_default_cartridge_cfg(srv_name)

